src/win/util.c

```diff
@@ -536,6 +536,24 @@ int uv_uptime(double* uptime) {
 }
 
 
+unsigned int uv_available_parallelism(void) {
+  SYSTEM_INFO info;
+  unsigned rc;
+
+  /* TODO(bnoordhuis) Should probably use GetLogicalProcessorInformationEx()
+   * to support systems with > 64 CPUs? On the other hand, uv_cpu_info() uses
+   * GetSystemInfo() too and so far no one has complained.
```

Comment:
Well there is a complaint from one user: https://github.com/libuv/libuv/pull/3458

Reply:
I'll update the comment. I don't think my win32 arcana fu is strong enough to transplant the logic from #3458 to this PR.